Cargo.lock

```diff
@@ -1351,8 +1351,8 @@ dependencies = [
 
 [[package]]
 name = "flatten-serde-json"
-version = "0.37.1"
-source = "git+https://github.com/meilisearch/milli.git?tag=v0.37.1#e1b2113cd9a467410841a254286e5e25961af43e"
+version = "0.37.2"
+source = "git+https://github.com/meilisearch/milli.git?tag=v0.37.2#1582b96119fedad39c726a6d4aeda0f53e868a3b"
```

Comment:
Not sure why but [the commit looks aligned with the v0.37.2 release](https://github.com/meilisearch/milli/releases/tag/v0.37.2) but why is it not the same as [the last commit of the main branch](https://github.com/meilisearch/milli/commits/main)?

Reply:
v0.37.2 of milli is not done on main, but on the `release-v0.37.2` branch, I wrote it on the release description 😇 
https://github.com/meilisearch/milli/releases/tag/v0.37.2

Reply:
Again, there are too many un-wanted PRs on `main` for v0.30.2 release

Reply:
You should check [this branch instead](https://github.com/meilisearch/milli/tree/release-v0.37.2)

Reply:
Whooops, indeed, therefore it looks good to me 👌